aomi/cli.py

```diff
@@ -235,7 +281,8 @@ def parser_factory(fake_args=None):
     """Return a proper contextual OptionParser"""
     parser = ArgumentParser(description='aomi')
     subparsers = parser.add_subparsers(dest='operation',
-                                       help='aomi operation')
+                                       help='Specify the data '
+                                       ' or extraction operation')
     extract_file_args(subparsers)
```

Comment:
Chaining won't work here, unless I wrap the `parser` in a class. I'm going to hold off on that until we refactor the `cli` module into some kinda class system https://github.com/Autodesk/aomi/issues/117